RELEASE.md

```diff
@@ -68,7 +68,7 @@ to reflect the next planned release, i.e.
 Using https://github.com/cilium/release, prepare the release notes between the
 last minor version (latest patch) and current.
 
-    ./release --base $LAST_RELEASE --head v$MAJOR.$MINOR
+    ./release --repo cilium/hubble --base v$LAST_RELEASE --head v$MAJOR.$MINOR
```

Comment:
I think I wrote this to intend `v` to be part of the `$LAST_RELEASE`, as in the latest tag that is known on that branch (but not documented it properly). If someone puts a `v` in their last release it still won't work :)

Either way works, really.